FinalXmL/FinalXmL/Pages/Index.cshtml.cs

```diff
@@ -9,34 +9,38 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using Newtonsoft.Json.Schema;
-using QuickType;
-using QuickType1;
+using QuickTypeCountry;
+using QuickTypeGeo;
 
 namespace FinalXmL.Pages
 {
     public class IndexModel : PageModel
     {
         public void OnGet()
         {
```

Comment:
Rather than putting the url string in the function call. I added a string variable in case the string needs to be manipulated and or called again. Overall in my opinion makes the code easier to read.